content/docs/rosa/vpn/_index.md

```diff
@@ -0,0 +1,189 @@
+    ```
+
+1. Import the client certificate
+     ```bash
+     CLIENT_CERT_ARN=$(aws acm import-certificate --certificate fileb://issued/aws.crt --private-key fileb://private/aws.key --certificate-chain fileb://ca.crt --region $REGION --query CertificateArn --output text)
+    ```
+
+## Create a Client VPN Endpoint
+
+   ```bash
+    VPN_CLIENT_ID=$(aws ec2 create-client-vpn-endpoint \
+    --client-cidr-block $VPN_CLIENT_CIDR \
+    --server-certificate-arn $SERVER_CERT_ARN \
+    --authentication-options Type=certificate-authentication,MutualAuthentication={ClientRootCertificateChainArn=$CLIENT_CERT_ARN} \
+    --connection-log-options Enabled=false --split-tunnel --query ClientVpnEndpointId --output text)
+   ```
+
+   Associate each private subnet with the client VPN endpoint
+   
+   ```bash
+   while IFS= read -r subnet;
+   do
+      echo "Associcating subnet '$subnet'"
+      aws ec2 associate-client-vpn-target-network --subnet-id $subnet --client-vpn-endpoint-id $VPN_CLIENT_ID
+   done <<< "$PRIVATE_SUBNET_IDS"
+   ```
+
+   Add an ingress authorization rule to a Client VPN endpoint
+
+   ```bash
+   aws ec2 authorize-client-vpn-ingress \
+    --client-vpn-endpoint-id $VPN_CLIENT_ID \
+    --target-network-cidr 0.0.0.0/0 \
+    --authorize-all-groups
+   ```
+
+## Configure your OpenVPN Client
+
+   ```bash
+   aws ec2 export-client-vpn-client-configuration --client-vpn-endpoint-id $VPN_CLIENT_ID --output text>client-config.ovpn
+   ```
+
+   * note: make sure you are still in the easy rsa / pki directory.
+
+   ```bash
+   echo '<cert>' >> client-config.ovpn
+   openssl x509 -in issued/aws.crt >> client-config.ovpn
+   echo '</cert>' >> client-config.ovpn
+   echo '<key>' >> client-config.ovpn
+   cat private/aws.key >> client-config.ovpn
+   echo '</key>' >> client-config.ovpn
+   ```
+## Add DNS Entries
+
+In order to resolve the ROSA Cluster domain name, you will need to either add the DNS server and the Route 53 Hosted Domain for the cluser to your VPN settings or /etc/hosts in machine you are connecting from.
+
+The DNS server will be the x.x.x.2 address of your VPC CIDR.  For example, if you VPC CIDR is 10.66.0.0/16 then your DNS server will be 10.66.0.2
+
+You can find the VPC ( machine ) CIDR with this command:
+```
+rosa describe cluster -c $ROSA_CLUSTER_NAME -o json | jq -r '.network.machine_cidr'
+```
+
+You can find the ROSA base domain with this command:
+```
+rosa describe cluster -c $ROSA_CLUSTER_NAME -o json | jq -r '.dns.base_domain'
+``` 
+## Configure your OpenVPN Client
+
+1. Import the client-config.ovpn file into your VPN Software.
+* Mac users - just double click the client-config.ovpn and it will be imported automatically into your VPN client.
+
+* Note: In order to connect to your cluster with the oc cli, you will need to update your OS DNS server with the DNS Server above after you connected to VPN.
+
+
+1. Connect your VPN.
+
+![screenshot of Vpn Connected](./images/connect-vpn-settings.png)
```

Comment:
indented.

Reply:
the screenshot image should be indented to show its inside the list item

Reply:
fixed